src/behavior.ts

```diff
@@ -114,5 +110,18 @@ export class Behavior implements IInteractiveBehavior {
                 !selected && hasHighlights
             );
         });
+
+        selection.attr("aria-selected", (dataPoint: HierarchyRectangularNode<SunburstDataPoint>) => {
+            return dataPoint.data.selected;
+        });
+
+        selection.on("keydown", (e, d: HierarchyRectangularNode<SunburstDataPoint>) => {
```

Comment:
Extract to `bindEvents` sections

Reply:
done: [13f944f](https://github.com/microsoft/powerbi-visuals-sunburst/pull/95/commits/13f944f4a01d13ab0d0643a6646d191a61a7e2ca)